### ISSUE TITLE
File to directory change is reported as: relaxed permissions from 0644 to 0755

### ISSUE THREAD

**Issue by hroncok** [rpminspect/rpminspect#1030]
**Describe the bug**
When a file turns into a directory, rpminspect reports "relaxed permissions from 0644 to 0755" -- this is confusing for the reviewer and should report "turned from a file to a directory" instead.

See https://artifacts.dev.testing-farm.io/0577cb41-f8fb-4e6a-b576-349ffd87e876/ from https://bodhi.fedoraproject.org/updates/FEDORA-2023-9e24a9d0b1

```
VERIFY:  /usr/lib64/python3.11/site-packages/TBB-0.1-py3.11.egg-info relaxed permissions from 0644 to 0755 on x86_64
```

**To Reproduce**

Version information for libraries and programs used by rpminspect as well as storage requirements. This result is for informational and diagnostic purposes only.

```
rpminspect version 1.11
zlib version 1.2.12
libmagic version 542
libclamav version 0.103.7
librpm version 4.18.0
libxml version 21003
json-c version 0.16
libcurl version 7.85.0 (OpenSSL/3.0.5 zlib/1.2.12 brotli/1.0.9 libidn2/2.3.4 libpsl/0.21.1 (+libidn2/2.3.3) libssh/0.10.4/openssl/zlib nghttp2/1.51.0)
libarchive version 3.6.1 (zlib/1.2.12 liblzma/5.2.5 bz2lib/1.0.8 liblz4/1.9.3 libzstd/1.5.2)
libyaml version 0.2.5
OpenSSL version 3.0.5 5 Jul 2022
xmlrpc-c version 1.51.8
msgunfmt (GNU gettext-tools) 0.21.1
annocheck version 10.93.
abidiff version 2.2.0
kmidiff version 2.2.0

Space required to download artifacts: 142872873 bytes (136 MiB)
Space required to unpack artifacts: 1965530869 bytes (2 GiB)
```

Command line arguments used to invoke rpminspect.

```
/usr/bin/rpminspect -c /usr/share/rpminspect/fedora.yaml --workdir /var/ARTIFACTS/work-rpminspectijy2x05g/rpminspect/discover/default-0/tests/workdir/ --format=json --output=/var/ARTIFACTS/work-rpminspectijy2x05g/rpminspect/execute/data/rpminspect/data/result.json --verbose --release=fc38 --profile=rawhide tbb-2020.3-12.fc37 96257509
```

**Expected behavior**

```
VERIFY:  /usr/lib64/python3.11/site-packages/TBB-0.1-py3.11.egg-info changed from a file to a directory
```

Note that the other way around should be BAD:

```
BAD:  /usr/lib64/python3.11/site-packages/TBB-0.1-py3.11.egg-info changed from a directory to a file, this will break RPM
```

**Environment:**
 - Fedora CI

**Comment by jwakely** [comment]
Nice, thanks!
